src/main/kotlin/commands/CapeCommand.kt

```diff
@@ -0,0 +1,463 @@
+                            message.error("Couldn't find a Cape User with the ID `$finalID`!")
+                            return@doesLaterIfHas
+                        }
+
+                        val cape = user.capes.find { it.capeUUID.equals(capeUUID, true) } ?: run {
+                            message.error(capeError(capeUUID))
+                            return@doesLaterIfHas
+                        }
+
+                        user.deleteCape(cape)
+
+                        message.success("Removed Cape `$capeUUID` from Cape User `$finalID`!")
+                    }
+                }
+            }
+        }
+
+        literal("list") {
+            doesLater {
+                val userCapes = message.getCapes() ?: return@doesLater
+
+                message.channel.send {
+                    embed {
+                        userCapes.forEach {
+                            val playerName = UUIDManager.getByUUID(it.playerUUID)?.name ?: "Not attached"
+                            field("Cape UUID ${it.capeUUID}", "Player Name: $playerName\nCape Type: ${it.type.realName}")
+                        }
+                        color = Colors.primary
+                    }
+                }
+            }
+        }
+
+        literal("attach") {
+            string("uuid") {
+                greedyString("username") {
+                    doesLater { context ->
+                        val username: String = context arg "username"
+                        val capeUUID: String = context arg "uuid"
+
+                        val capes = message.getCapes() ?: return@doesLater
+
+                        val cape = capes.find { it.capeUUID == capeUUID } ?: run {
+                            message.error(capeError(capeUUID))
+                            return@doesLater
+                        }
+
+                        val profilePair = UUIDManager.getByString(username)
+
+                        val msg = if (profilePair != null) {
+                            message.normal("Found UUID to attach to Cape `$capeUUID` - verifying")
+                        } else {
+                            message.channel.send {
+                                embed {
+                                    title = "Error"
+                                    description = "Couldn't find an account with the UUID/Name `$username`!\n" +
+                                            "Make sure you have a real Mojang account and with correct UUID/Name, " +
+                                            "contact a moderator if this keeps happening."
+                                    color = Colors.error
+                                }
+                            }
+                            return@doesLater
+                        }
+
+                        var alreadyAttached: Cape? = null
+                        var alreadyAttachedUser: Long? = null
+
+                        for (capeUser in capeUserMap) {
```

Comment:
You can specific name for key and value for an entry. Just replace `capeUser` with `(id, capes)`
```suggestion
                        for ((id, capes) in capeUserMap) {
```